### DISCUSSION TITLE
slidespercolumns: 'column' doesn't work

### DISCUSSION THREAD

**Discussion by mojifier** [nolimits4web/swiper#4573] [Q&A]
I check this parameter `slidesPerColumnFill: 'column'` for show slider like this picture:

![meTC1](https://user-images.githubusercontent.com/19752074/118712034-87706680-b835-11eb-8ebb-9e907a066690.png)

1 | 3 | 5 | 7 | 9
2 | 4 | 6 | 8 | 10

But in action it's doesn't work. how do can I fix this problem?

demo [here](https://jsfiddle.net/rwnpuvs4/1/)

**Comment by Dupflo** [comment]
Hi, 

I got similar issue and after investigating and telling myself that there is an issue with Swiper I finally found a solution...

You must set an fix height to the slider and also the slide to make you stuff working

https://jsfiddle.net/dupflo/qmocL9b2/21/

**Reply by mojifier** [reply]
Hi,

I your code, you sort slide in `row` format. 

```
1 | 2 | 3 | 4 | 5
6 | 7 | 8 | 9 | 10

```

I need to show slider in `column` format like this:

```

1 | 3 | 5 | 7 | 9
2 | 4 | 6 | 8 | 10

```
